frontend/src/pages/modelRegistry/screens/components/DeployRegisteredModelModal.tsx

```diff
@@ -122,7 +124,7 @@ const DeployRegisteredModelModal: React.FC<DeployRegisteredModelModalProps> = ({
         isOpen
         onClose={() => onClose(false)}
         actions={[
-          <Button key="deploy" variant="primary" onClick={handleSubmit}>
+          <Button key="deploy" variant="primary" onClick={handleSubmit} isDisabled>
```

Comment:
Wait, why is this button always disabled?

Edit: ah, I see - it's because this particular return of the Modal only happens when there's not a valid selected project, otherwise we'll render the `ManageKServeModal` or `ManageInferenceServiceModal`... ok. Can we add a comment about that?

Reply:
Followed what was in #3074 .. this removed during the PF6 refactors somehow... I just reinstated this. I'll add a comment to remove any suspicions.